docs/rules/grouped-accessor-pairs.md

```diff
@@ -0,0 +1,324 @@
+# Require grouped accessor pairs in object literals and classes (grouped-accessor-pairs)
+
+Getter and setter for the same property don't necessarily have to be defined one after the other.
```

Comment:
```suggestion
A getter and setter for the same property don't necessarily have to be defined adjacent to each other.
```

Reply:
Thanks! Had trouble with the opening sentence